ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/DataStructureUtil.java

```diff
@@ -1587,6 +1587,16 @@ public SignedBlsToExecutionChange randomSignedBlsToExecutionChange() {
         .create(randomBlsToExecutionChange(), randomSignature());
   }
 
+  public Bytes32 randomBlsWithdrawalCredentials() {
+    return Bytes32.wrap(
+        Bytes.concatenate(Bytes.fromHexString("0x00"), randomBytes32().slice(0, 31)));
+  }
+
+  public Bytes32 randomEth1WithdrawalCredentials() {
+    return Bytes32.wrap(
+        Bytes.concatenate(Bytes.fromHexString("0x01"), randomBytes32().slice(0, 31)));
+  }
```

Comment:
eth1 withdrawal credentials are slightly different...
```
  private static final Bytes ETH1_WITHDRAWAL_PREFIX = Bytes.fromHexString("0x01");
  private static final Bytes WITHDRAWAL_BUFFER = Bytes.repeat((byte) 0x00, 11);
  public static Bytes32 withdrawalAddressFromEth1Address(final Bytes20 toExecutionAddress) {
    return Bytes32.wrap(
        Bytes.concatenate(
            ETH1_WITHDRAWAL_PREFIX, WITHDRAWAL_BUFFER, toExecutionAddress.getWrappedBytes()));
  }
```
above is a snippet from my current work, where I'll create an eth1 address, we can probably migrate this into helpers after, but maybe for this one just make the `Bytes32.wrap(Bytes.fromHexString("0x010000000000000000000000"), randomEth1Address()))` or something...